starlite/testing.py

```diff
@@ -85,6 +88,25 @@ def __init__(
             backend_options=backend_options,
         )
 
+    def __enter__(self: T_client, *args: Any, **kwargs: Any) -> T_client:
+        """
+        Starlette's `TestClient.__enter__()` return value is strongly typed to return their own
+        `TestClient`, i.e., not-generic to support subclassing.
+
+        Override here to provide a nicer typing experience downstream from us.
```

Comment:
from us -> for us

Reply:
Well I actually do mean "from" us, i.e., for our downstream users. I'll rephrase.